.env.development

```diff
@@ -1,7 +1,7 @@
 EXTEND_ESLINT=true # Set linting to true on dev
 # developemnt API and keys
-REACT_APP_API_HOST='https://wmca-api-portal-staging.azure-api.net'
-REACT_APP_API_KEY='0d4cca4a2c5d40c3bfbbfe45d1bbf294'
+REACT_APP_API_HOST='https://wmnapi-alpha-01.azure-api.net'
+REACT_APP_API_KEY='fe4085ce404c4653b939d20a4ccbd581'
```

Comment:
If you could just change these vars back to:
`REACT_APP_API_HOST='https://wmca-api-portal-staging.azure-api.net'`
`REACT_APP_API_KEY='0d4cca4a2c5d40c3bfbbfe45d1bbf294'`